course6/toyapp/displaylib/vector.cc

```diff
@@ -0,0 +1,13 @@
+#include "vector.h"
+
+namespace Display {
+
+Vector2D Vector3D::project() const
```

Comment:
The naming of the function is not consistent with the project's style, which uses CamelCasing

Reply:
My bad, I'm using lowerCamelCase in my home project and did this out of habit.